grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -27,4 +37,36 @@ class WpsService extends AsyncDownloadService {
     def onResponseSuccess = { resp, xmlReader ->
         return new groovy.xml.StreamingMarkupBuilder().bindNode(xmlReader).toString()
     }
+
+    def notifyViaEmail(params) {
```

Comment:
How's about I have the default email destination be the greenmail "server" - i.e. emails will not be sent.  chef can override this with a real SMTP destination (just as it already does with ggd) when the security is ready.  

My feel is that it would be better to do this with apache (or something else off-the-shelf) rather than code our own logic - but you may have a different opinion on this?